bundles/org.openhab.binding.konnected/src/main/java/org/openhab/binding/konnected/internal/handler/KonnectedHandler.java

```diff
@@ -56,7 +56,7 @@ public class KonnectedHandler extends BaseThingHandler {
     private final String konnectedServletPath;
     private final KonnectedHTTPUtils http = new KonnectedHTTPUtils(30);
     private String callbackIpAddress = null;
-    private String moduleIpAddress;
+    private String baseUri;
```

Comment:
To avoid compilation warnings, you could initialize it with an empty string.

Reply:
I'm not getting any warnings for this. I prefer to have it be uninitialized and have any bugs related to it not set be obvious.